airflow/providers/common/sql/hooks/sql.py

```diff
@@ -515,3 +516,40 @@ def test_connection(self):
             message = str(e)
 
         return status, message
+
+    def get_database_info(self, connection):
```

Comment:
Is this specific to lineage? If yes, would it make sense to mention this in the method name explicitly?

It is probably worth adding a docstring.

Reply:
Does `:meta private:` make method to be skipped in docs generation?
If so we probably should follow this rule for all OpenLineage related methods or none of them.